modules/javafx.graphics/src/main/java/javafx/scene/layout/VBox.java

```diff
@@ -230,8 +231,8 @@ public VBox(double spacing) {
     }
 
     /**
-     * Creates an VBox layout with spacing = 0.
-     * @param children The initial set of children for this pane.
+     * Creates n {@code VBox} layout with {@code spacing = 0}.
```

Comment:
oops:  n -> a

Also, you might look through the file for "an vbox" since it occurs in other places.